requirements/edx/base.txt

```diff
@@ -851,9 +854,9 @@ random2==1.0.1
     # via -r requirements/edx/base.in
 recommender-xblock==2.0.1
     # via -r requirements/edx/base.in
-redis==3.5.3
+redis==4.0.0
```

Comment:
`redis` jumps a major version with this, might break

Reply:
No breaking change mentioned in the release notes: https://github.com/redis/redis-py/releases/tag/v4.0.0